palace/main.cpp

```diff
@@ -60,9 +60,6 @@ int main(int argc, char *argv[])
   int world_size = Mpi::Size(world_comm);
   Mpi::Print(world_comm, "\n");
 
-  // Initialize timer.
-  Timer timer;
```

Comment:
I also like the idea, but thinking more I am not sure if it will mean much until we have clearer definition of what the existing categories should cover. `OTHER` may end up capturing very little given we have large blocks of code captured by existing categories, for example I believe all lines of code between the mesh refinement and solve is counted as operator construction. Thoughts?

Reply:
I would keep it as `INIT` and just move the first `TimerBlock` construction to be the first line of `main`. This way we can actually capture the entire initialization including MPI initialization, config file parsing, etc. If we eventually want to separate that from reading the mesh file that's fine but for now that seems like the most logical approach.